requests/cookies.py

```diff
@@ -13,6 +13,8 @@
 import time
 import calendar
 import collections
+
+from . import utils
```

Comment:
Odd, I didn't experience any issues during the test run. Perhaps it only appears if you import the cookies.py module directly.

Anyways, that sounds like a fine solution to me. I'll work on a patch later today to incorporate it into this pull request.

Thanks for the quick feedback.